validator/accounts/wallet/wallet.go

```diff
@@ -290,6 +292,25 @@ func (w *Wallet) InitializeKeymanager(ctx context.Context, cfg iface.InitKeymana
 		if err != nil {
 			return nil, errors.Wrap(err, "could not initialize remote keymanager")
 		}
+	case keymanager.Web3Signer: //option should not work for --web
+		configFile, err := w.ReadKeymanagerConfigFromDisk(ctx)
+		if err != nil {
+			return nil, errors.Wrap(err, "could not read web3signer keymanager config")
+		}
+		config, err := remote_web3signer.UnmarshalConfigFile(configFile)
+		if err != nil {
+			return nil, errors.Wrap(err, "could not unmarshal web3signer keymanager config file")
+		}
+		// injected in the validator runner
+		if len(cfg.GenesisValidatorsRoot) != 0 {
```

Comment:
Simplify by checking if len(cfg.GenesisValidatorsRoot) != 32 { return nil, errors.New("genesis validators root value is required for web3signer keymanager but was not provided" }